### PR TITLE
Add linters

### PR DESCRIPTION
**Pull request by TahenyBELGUITH**
in this project:
 -I install the linters  and I check it

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
Comment (carlos-ssh) [inline .github/workflows/linters.yml:1]: **Check linter:**

- Linter should be set up correctly for this repository.
- Linter result should be green. AKA: Linter should report NO errors.
- There should be no custom changes to linter config (as an attempt to cheat linter error reports).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
```
Comment (carlos-ssh) [inline index.html:11]: - A .html file should include a header with the text "Hello Microverse!". Use h1 tag for that.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @TahenyBELGUITH , 

Good job so far!
There are some issues that you still need to work on to go to the next project, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (TahenyBELGUITH) [reply index.html]: done

File: .github/workflows/linters.yml (hunk 1 above)
Comment (TahenyBELGUITH) [reply .github/workflows/linters.yml:1]: done

**Review by carlos-ssh — APPROVED** [review]
Hi @TahenyBELGUITH, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**